### PR TITLE
Integrations to packs - Phase 1 (Dan)

### PR DESCRIPTION
**Pull request by dantavori**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Moved the following integrations from Legacy pack to separate packs:
- Accessdata
- ANYRUN
- ARIAPacketIntelligence
- AWS-ACM
- AWS-CloudTrail
- AWS-EC2
- AWS-IAM
- AWS-Lambda
- AWS-S3
- AWS_DynamoDB

### CODE REVIEW

**Comment by dantavori** [comment]
@yuvalbenshalom because weird duplications like the following (see first and last lines) which caused validate to fail:
image

File: Packs/ANYRUN/pack_metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "ANY.RUN",
+    "description": "ANY.RUN is a cloud-based sanbox with interactive access.",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/ANYRUN/pack_metadata.json:5]: please remove

File: Packs/ANYRUN/pack_metadata.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        "Forensics & Malware Analysis"
+    ],
+    "tags": [
+        "ANY.RUN"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/ANYRUN/pack_metadata.json:20]: please remove

File: Packs/ARIAPacketIntelligence/pack_metadata.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "ARIA Packet Intelligence",
+    "description": "The ARIA Cybesecurity Solutions Software-Defined Security (SDS)  platform integrates with Demisto to add robustness when responding to incidents. The combination of ARIA hardware, in the form of a Secure Intelligent Adapter (SIA), and software, specifically Packet Intelligence and SDS orchestrator (SDSo), provides the elements required to react instantly when an incident is detected. When integrated with the ARIA solution, you can create playbooks that instruct one or more SIAs to add, modify, or delete rules automatically. These rule changes, which take effect immediately, can block conversations, redirect packets to a recorder or VLAN, or perform a variety of other actions.",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/ARIAPacketIntelligence/pack_metadata.json:5]: please remove

File: Packs/ARIAPacketIntelligence/pack_metadata.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        "Network Security"
+    ],
+    "tags": [
+        "ARIA Packet Intelligence"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/ARIAPacketIntelligence/pack_metadata.json:20]: please remove

File: Packs/AWS-ACM/pack_metadata.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "AWS - ACM",
+    "description": "Amazon Web Services Certificate Manager Service (acm)",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/AWS-ACM/pack_metadata.json:5]: please remove

File: Packs/AWS-ACM/pack_metadata.json [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+        "IT Services"
+    ],
+    "tags": [
+        "AWS - ACM"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/AWS-ACM/pack_metadata.json:20]: please remove

File: Packs/AWS-CloudTrail/pack_metadata.json [hunk 7]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "AWS - CloudTrail",
+    "description": "Amazon Web Services CloudTrail.",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/AWS-CloudTrail/pack_metadata.json:5]: please remove

File: Packs/AWS-CloudTrail/pack_metadata.json [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+        "IT Services"
+    ],
+    "tags": [
+        "AWS - CloudTrail"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/AWS-CloudTrail/pack_metadata.json:20]: please remove

File: Packs/AWS-EC2/pack_metadata.json [hunk 9]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "AWS - EC2",
+    "description": "Amazon Web Services Elastic Compute Cloud (EC2)",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/AWS-EC2/pack_metadata.json:5]: please remove

File: Packs/AWS-EC2/pack_metadata.json [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+        "IT Services"
+    ],
+    "tags": [
+        "AWS - EC2"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/AWS-EC2/pack_metadata.json:20]: please remove

File: Packs/AWS-IAM/pack_metadata.json [hunk 11]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "AWS - IAM",
+    "description": "Amazon Web Services Identity and Access Management (IAM)",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/AWS-IAM/pack_metadata.json:5]: please remove

File: Packs/AWS-IAM/pack_metadata.json [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+        "IT Services"
+    ],
+    "tags": [
+        "AWS - IAM"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/AWS-IAM/pack_metadata.json:20]: please remove

File: Packs/AWS-Lambda/pack_metadata.json [hunk 13]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "AWS - Lambda",
+    "description": "Amazon Web Services Serverless Compute service (lambda)",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/AWS-Lambda/pack_metadata.json:5]: please remove

File: Packs/AWS-Lambda/pack_metadata.json [hunk 14]
```diff
@@ -0,0 +1,25 @@
...
+        "IT Services"
+    ],
+    "tags": [
+        "AWS - Lambda"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/AWS-Lambda/pack_metadata.json:20]: please remove

File: Packs/AWS-S3/pack_metadata.json [hunk 15]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "AWS - S3",
+    "description": "Amazon Web Services Simple Storage Service (S3)",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/AWS-S3/pack_metadata.json:5]: please remove

File: Packs/AWS-S3/pack_metadata.json [hunk 16]
```diff
@@ -0,0 +1,25 @@
...
+        "IT Services"
+    ],
+    "tags": [
+        "AWS - S3"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/AWS-S3/pack_metadata.json:20]: please remove

File: Packs/AWS_DynamoDB/pack_metadata.json [hunk 17]
```diff
@@ -0,0 +1,25 @@
...
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/AWS_DynamoDB/pack_metadata.json:5]: please remove

File: Packs/AWS_DynamoDB/pack_metadata.json [hunk 18]
```diff
@@ -0,0 +1,25 @@
...
+        "IT Services"
+    ],
+    "tags": [
+        "Amazon DynamoDB"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/AWS_DynamoDB/pack_metadata.json:20]: please remove

File: Packs/Accessdata/pack_metadata.json [hunk 19]
```diff
@@ -0,0 +1,25 @@
+{
+    "name": "Accessdata",
+    "description": "Use the Accessdata integration to protect against and provide additional visibility into phishing and other malicious email attacks.",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/Accessdata/pack_metadata.json:5]: please remove

File: Packs/Accessdata/pack_metadata.json [hunk 20]
```diff
@@ -0,0 +1,25 @@
...
+        "Forensics & Malware Analysis"
+    ],
+    "tags": [
+        "Accessdata"
+    ],
+    "created": "2020-04-14T00:00:00Z",
+    "updated": "2020-03-14T00:00:00Z",
+    "beta": false,
+    "deprecated": false,
+    "certification": "verified",
```
Comment (yuvalbenshalom) [inline Packs/Accessdata/pack_metadata.json:20]: please remove

File: Packs/AWS-IAM/pack_metadata.json [hunk 21]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "AWS - IAM",
+    "description": "Amazon Web Services Identity and Access Management (IAM)",
+    "support": "xsoar",
+    "serverMinVersion": "1.0.0",
```
Comment (yuvalbenshalom) [inline Packs/AWS-IAM/pack_metadata.json:5]:
```suggestion
    "serverMinVersion": "1.0.0",
```

Comment (dantavori) [reply Packs/AWS-IAM/pack_metadata.json:5]:
```suggestion
```

Comment (dantavori) [reply Packs/AWS-IAM/pack_metadata.json:5]: missed it FSR... besides that the build passed

**Pull request merged**